lib/rules/landmark-at-least-one-main.json

```diff
@@ -0,0 +1,16 @@
+{
+  "id": "landmark-at-least-one-main",
+  "selector": "html",
+  "tags": [
+	"cat.best-practice"
+  ],
+  "metadata": {
+    "description": "Each page must have at least one main landmark, used to identify the main content.",
+    "help": "A <main> element or an element with a role='main' attribute defines a main landmark."
```

Comment:
I should have mentioned this earlier, but the `help` text is what shows up in the sidebar of the aXe extensions. So it should be short, and represent the violation a user will encounter when the rule fails. Something like "Page should contain at least one main landmark" ?